optimum/amd/ryzenai/modeling.py

```diff
@@ -623,6 +624,48 @@ def forward(self, pixel_values):
 
         return ImageClassifierOutput(logits=next(iter(outputs.values())))
 
+    @classmethod
+    def _export(
+        cls,
+        model_id: str,
+        config: "PretrainedConfig" = None,
+        use_auth_token: Optional[Union[bool, str]] = None,
+        revision: Optional[str] = None,
+        force_download: bool = False,
+        cache_dir: Optional[str] = None,
+        subfolder: str = "",
+        # use_auth_token=use_auth_token,
+        local_files_only: bool = False,
+        trust_remote_code: bool = False,
+        vaip_config: Optional[str] = None,
+        # vaip_config=vaip_config,
+        provider: str = "CPUExecutionProvider",
+        session_options: Optional[ort.SessionOptions] = None,
+        provider_options: Optional[Dict[str, Any]] = None,
+        use_io_binding: Optional[bool] = None,
+        task: Optional[str] = None,
+    ) -> "RyzenAIModel":
+
+        save_dir = TemporaryDirectory()
+        save_dir_path = Path(save_dir.name)
+        main_export(
+            model_name_or_path=model_id,
+            output=save_dir_path,
+            task="image-classification",
+            opset=17,
+            batch_size=1,
+            no_dynamic_axes=True,
+        )
+        config = PretrainedConfig.from_pretrained(save_dir_path)
```

Comment:
The config is provided in the method, lets avoid overriding it.

Reply:
Done